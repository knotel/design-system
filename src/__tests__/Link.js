import React from 'react'
import renderer from 'react-test-renderer'
import { Link } from '..'

describe('Link', () => {
  test('renders', () => {
    const json = renderer
      .create(<Link>Dummy</Link>, { disableLifecycleMethods: true })
      .toJSON()
    expect(json).toMatchSnapshot()
  })
})
