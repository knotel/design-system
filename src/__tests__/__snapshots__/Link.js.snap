// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders 1`] = `
<a
  className="neiLn"
  color="blue"
>
  Dummy
</a>
`;
